.circleci/config.yml

```
@@ -18,22 +18,36 @@ commands:
           paths:
             - node_modules
           key: v1-dependencies-{{ checksum "package.json" }}
+  test-clearhaus:
+    description: "test"
+    steps:
+      - prepare
+      - run:
+          name: Run tests
+          command: npm run test-clearhaus
   test:
     description: "test"
     steps:
       - prepare
       - run:
           name: Run tests
           command: npm run test
+  test-intergiro:
+    description: "test Intergiro"
+    steps:
+      - prepare
+      - run:
+          name: Run Intergiro tests
+          command: npm run test-intergiro
 jobs:
   build:
     <<: *default
     steps:
-      - test
-  check:
+      - test-clearhaus
```

Should be test? We should run all test on build right? So that we know they work before merging.
```suggestion
      - test
```

---

change this to test